packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-write-execution.js

```diff
@@ -4,6 +4,8 @@ const test = require('ava');
 const cryptoRandomString = require('crypto-random-string');
 const sinon = require('sinon');
 const uuidv4 = require('uuid/v4');
+const { removeNilProperties } = require('@cumulus/common/util');
+
```

Comment:
I think you can remove that import and this newline

Reply:
69b7df972